### PR TITLE
[ISSUE#1750] The method init() in DumpService is too long to read easily

### PR DESCRIPTION
**Pull request by shirenchuang**
## What is the purpose of the change

DumpService 初始化的方法 精简了一下; 把一些方法单独提了出来; 让init看起来更容易阅读一点

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/main/java/com/alibaba/nacos/config/server/manager/TaskManager.java [hunk 1]
```diff
@@ -76,7 +76,10 @@ public void run() {
     public TaskManager() {
```
Comment (chuntaojun) [inline config/src/main/java/com/alibaba/nacos/config/server/manager/TaskManager.java]: 这个修复先移除吧，一个issue对应一个pr，这个修复换到另一个pr去做吧

File: config/src/main/java/com/alibaba/nacos/config/server/service/dump/DumpService.java [hunk 2]
```diff
@@ -210,30 +213,14 @@ private void dumpConfigInfo(DumpAllProcessor dumpAllProcessor)
...
-                            ConfigInfoWrapper configInfo = persistService.queryConfigInfo(dataId, group, tenant);
-                            ConfigService.dumpChange(dataId, group, tenant, configInfo.getContent(),
-                                configInfo.getLastModified());
-                        }
-                        LogUtil.defaultLog.error("end checkMd5Task");
-                    }
-                };
-                TimerTaskService.scheduleWithFixedDelay(checkMd5Task, 0, 12,
+
+                dumpChangeProcessor.process(DumpChangeTask.TASK_ID,new DumpChangeTask());
```
Comment (nkorange) [inline config/src/main/java/com/alibaba/nacos/config/server/service/dump/DumpService.java]: Format code please.

**Comment by nkorange** [comment]
Related issue is https://github.com/alibaba/nacos/issues/1750

**Comment by KomachiSion** [comment]
No response and update for a long time, close PR, if wanted resolve issue, please re-submit PR, thanks.

**Pull request closed without merging**